Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -0,0 +1,242 @@
+            indicators.extend(self.http_request(feed_type="URL Feed"))
+
+        parsed_indicators = []
+
+        for indicator in indicators:
+            if indicator:
+                indicator_type = self.find_indicator_type(indicator)
+
+                parsed_indicators.append({
+                    "type": indicator_type,
+                    "value": indicator,
+                    "rawJSON": {
+                        'value': indicator,
+                        'type': indicator_type
+                    }
+                })
+
+        return parsed_indicators
+
+
+def module_test_command(client: Client, args: dict):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: Autofocus Feed client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    indicator_feeds = client.indicator_feeds
+    if 'Daily Threat Feed' in indicator_feeds:
+        client.indicator_feeds = ['Daily Threat Feed']
+        try:
+            client.build_iterator()
+        except Exception:
+            raise Exception("Could not fetch Daily Threat Feed\n\nCheck your API key and your connection AutoFocus.")
+
+    if 'URL Feed' in indicator_feeds:
+        client.indicator_feeds = ['URL Feed']
+        try:
+            client.build_iterator()
+        except Exception:
+            raise Exception(f"Could not fetch URL Feed {client.url_feed_suffix}\n\n"
+                            f"Check your API key the URL Feed ID and Name and Check if they are Enabled in AutoFocus.")
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: dict):
+    """Initiate a single fetch-indicators
+
+    Args:
+        client(Client): The AutoFocus Client.
+        args(dict): Command arguments.
+
+    Returns:
+        str, dict, list. the markdown table, context JSON and list of indicators
+    """
+    indicators = client.build_iterator()
+    if args.get('offset'):
+        indicators = indicators[int(str(args.get('offset'))):]
+
+    if args.get('limit'):
+        indicators = indicators[:int(str(args.get('limit')))]
+
+    hr_indicators = []
+    for indicator in indicators:
+        hr_indicators.append({
+            "Value": indicator.get('value'),
+            "Type": indicator.get('type')
+        })
+
+    human_readable = tableToMarkdown("Indicators from AutoFocus:", hr_indicators,
+                                     headers=['Value', 'Type'], removeNull=True)
+
+    if args.get('limit'):
+        human_readable = human_readable + f"\nTo bring the next batch of indicators run:\n!get-indicators " \
+            f"limit={args.get('limit')} offset={int(str(args.get('limit'))) + int(str(args.get('offset')))}"
+
+    return human_readable, {
+        f"{SOURCE_NAME}.Indicator(val.Value === obj.Value && val.Type === obj.Type)": hr_indicators
+    }, indicators
```

Comment:
are we sure that adding to context is a good thing?

Reply:
will delete context